src/engine.js

```diff
@@ -9,62 +9,27 @@ const loadTasksFromLocalStorage = () => {
   }
 };
 
-// Save and update local storage
-const saveTasksToLocalStorage = () => {
-  localStorage.setItem('todoList', JSON.stringify(list));
-  const filteredList = list.filter((task) => !task.completed);
-  localStorage.setItem('todoList', JSON.stringify(filteredList));
-};
-
-// Display tasks
-const displayItemsByIndex = () => {
-  listContainer.innerHTML = '';
-
+// Event listeners for the checkboxes and delete buttons
+const attachEventListeners = () => {
   list.forEach((item) => {
-    const { description, completed, index } = item;
-
-    const listItem = document.createElement('div');
-    const descriptionSpan = document.createElement('span');
-    const completedRadio = document.createElement('input');
-    const edit = document.createElement('button');
-    const hr = document.createElement('hr');
-
-    edit.classList.add('edit-task');
-
-    listItem.classList.add('icon');
-
-    descriptionSpan.textContent = description;
-    completedRadio.type = 'checkbox';
-    completedRadio.name = `completed_${index}`;
-    completedRadio.checked = completed;
-    edit.textContent = 'Edit task';
+    const { index } = item;
+    const completedRadio = document.querySelector(`input[name="completed_${index}"]`);
 
-    // Handle checkbox change event
     completedRadio.addEventListener('change', () => {
       item.completed = completedRadio.checked;
       saveTasksToLocalStorage();
     });
-
-    edit.addEventListener('click', () => {
-      const newDescription = prompt('Enter the new description:');
-      if (newDescription !== '') {
-        editTaskDescription(index, newDescription);
-      } else {
-        alert('description must be at least two chars');
-      }
-    });
-
-    listItem.appendChild(completedRadio);
-    listItem.appendChild(edit);
-    listItem.appendChild(descriptionSpan);
-    listItem.appendChild(hr);
-
-    listContainer.appendChild(listItem);
-
-    document.getElementById('taskInput').value = '';
   });
 };
 
+const Delete = document.getElementById('clear');
+
+Delete.addEventListener('click', () => {
+  list = list.filter((item) => !item.completed);
+  saveTasksToLocalStorage();
+  displayItemsByIndex();
+});
```

Comment:
- [ ] The code is well for "Clear All Completed Tasks" we are going to use it for the next milestone but now we need to delete a specific task as we can see 👀 in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Do you see when the user clicks on the three vertical points and the trash icon appears? Please 🙏🏽 create the trash icon to delete the task with the same behavior to be similar to the wireframe.